core/templates/dev/head/collection_editor/collection_editor_save_publish_directive.html

```diff
@@ -0,0 +1,61 @@
+<script type="text/ng-template" id="inline/collection_editor_save_publish_directive">
+  <div>
+    <div class="nav oppia-navbar-nav">
+      <h2>
+        <[getCollection().getTitle()]> (v<[getCollection().getVersion()]>)
+
+        <button class="btn btn-default oppia-save-draft-button"
+                ng-class="{'btn-success': isCollectionSaveable()}"
+                ng-click="saveCollection('')"
```

Comment:
Implemented a modal to ensure a commit message is specified when it needs to be. This was done in #1989.